### PR TITLE
Add New Style based on old Newspaper

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds a new style pack based on https://github.com/Automattic/newspack-issues/issues/68

Heading font: [Old Standard TT](https://fonts.google.com/specimen/Old+Standard+TT)
Body font: [EB Garamond](https://fonts.google.com/specimen/EB+Garamond)

I tried to keep the changes as minimal as possible whilst offering an alternate style to the theme 😊

### How to test the changes in this Pull Request:

1. Go to the customizer and select "Style 5"

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: sass/styles/style-5/variables-style/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Font and typographic variables
+
+$font__heading: "Old Standard TT", Georgia, serif;
+$font__body: "EB Garamond", Georgia, serif;
+
+$font__size_base: 20px;
+$font__size_base_sm: 16px;
+$font__size-ratio: 1.2;
+
+$font__size-xxxs:  1em * 0.5; // 10px
```
[the remark this answers is not in the document]
Comment (thomasguillot) [reply sass/styles/style-5/variables-style/_fonts.scss]: Thank you @laurelfulford! I _think_ I took care of everything 😬

**Comment by thomasguillot** [comment]
Note: This PR has now been rebased against `master`

**Comment by laurelfulford** [comment]
Thanks for the updates, Thomas! 

The only thing I spotted that's still not working correctly is a bit of the custom typography behaviour:

> * A few things aren't picking up the custom header styles -- like the 'accent headers', categories, section headers for the article block, and dropcaps:

Dropcaps look good on the front-end, but don't use the same font in the editor:

Front end: 
image

Editor:
image

The 'Section headers' in the Article block are using the body type when you use custom fonts (here I have Chalkboard for the headers, and Arial for the body):

image

This is fine, but there's a class, `.accent-header`, that should look the same, but currently uses the header font:

image

---

I copied the rest of the feedback below to make sure I didn't miss anything:

> * The Pullquotes don't inherit any of the custom fonts -- in this screenshot I have Chalkboard set for the headers, and Arial set for the body.

This is resolved!

> * We've added a custom style to the Columns block that adds a border; it's the only one that's not using the style pack's stronger dark grey (vertical line here):

Looks good! 

> * What do you think about changing the separator blocks with this style pack? Specifically wondering how double lines would look for the first two, or something similar? Or am I just over-complicating things?

Thanks! I think this a nice touch. 

> * The dropdown menus have been updated to always use dark backgrounds; the background is now on the links, rather than the `ul`, so that little triangle thing can be added on the top:

Looks good!

> * Each level of flyout after the first one also moves down a pixel, rather than being vertically aligned.

Looks good!

> * When the 'short' header is set, the primary menu feels a bit large:

Looks good!

> * This is one of the theme's more buried features: there's an option to turn off the all caps used in some of the style pack accents, under Customizer > Typography -- it's mostly intended for the style packs that are heavy on the all-caps text (1, 2 and 3). For the default and style 4, it's lower key -- I think it would be enough to piggy back off of the selectors there:

Thanks for adding this! 

> * For individual posts, there's an option under the featured image to display the featured image behind or beside the post title -- when that's enabled, I think the border below the menu should be hidden:

This looks good! 

> * When the 'beside' option is selected, there is also a lot of space between the author name and date: 

Looks good!

> * In some cases, the headers with the border next to them get a bit clipped vertically:

Looks good!

**Comment by laurelfulford** [comment]
This all looks good, @thomasguillot!

**Pull request merged**